### PR TITLE
[ENH] Use palette colors in more places

### PR DESCRIPTION
**Pull request by ales-erjavec**
##### Issue

Some visualizations use hardcoded colors and do not account for the effective color palette.

##### Description of changes

Replace use of hardcoded colors with palette colors.  

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by janezd — APPROVED** [review]

**Comment by ajdapretnar** [comment]
The only fault I can find here is the Correspondence Analysis suddenly showing no points at all. 🤷‍♀️



The rest is ok. I hope I have tested it well, I wasn't entirely sure what to check. 😅

File: Orange/widgets/visualize/owscatterplotgraph.py [hunk 1]
```diff
@@ -91,6 +91,31 @@ def clear(self):
...
+            color.setAlpha(100)
+            pen = QPen(color, 1)
+            pen.setCosmetic(True)
+            return pen
+
+    def brush(self):
+        if self.__brush is not None:
+            return QBrush(self.__brush)
+        else:
+            color = self.palette().color(QPalette.Window)
```
Comment (janezd) [inline Orange/widgets/visualize/owscatterplotgraph.py:108]: I think this is too dark, at least on my machine. I don't have a better suggestion, except, perhaps, `super().brush()` (which, for me, apparently uses a transparent background).

Screenshot 2021-11-19 at 14 04 23

Comment (ales-erjavec) [reply Orange/widgets/visualize/owscatterplotgraph.py:108]: Strange its supposed to be: 
Screenshot 2021-11-19 at 14 29 02

Comment (ajdapretnar) [reply Orange/widgets/visualize/owscatterplotgraph.py:108]: I think I can finally say: Janez, your Qt is a mess. 😆 
(i.e. My version is ok, too.)

Comment (janezd) [reply Orange/widgets/visualize/owscatterplotgraph.py:108]: My `QPalette.Window` is black, which it indeed shouldn't be (in light mode on macOS). Any ideas why?

```
>>> hex(QPalette().color(QPalette.Window).rgb())
'0xff000000'
```

Comment (ales-erjavec) [reply Orange/widgets/visualize/owscatterplotgraph.py:108]: Can you post the contents of `~/.config/biolab.si/Orange.ini`in particular `[application-style]` section.

Comment (janezd) [reply Orange/widgets/visualize/owscatterplotgraph.py:108]:
```
[application-style]
palette=
style-name=
```

**Comment by janezd** [comment]
I tried in a new environment. It's still (transparent) black.

**Comment by VesnaT** [comment]
image

On:
- macOS Catalina
- python 3.8.3
- PyQt5==5.12.3

**Comment by janezd** [comment]
@ales-erjavec, I downloaded an Orange bundle, replaced the widgets directory with the files from this PR. The legend still has a black background. It looks as something between Qt and Catalina, not my installation of Qt.

**Comment by janezd** [comment]
As suggested by @markotoplak, I tried creating a fresh user. Still black.

**Comment by janezd** [comment]
Found it. Works with pyqtgraph 0.12 (the change that was made just to improve performance, it appears: https://github.com/pyqtgraph/pyqtgraph/pull/1569/commits/fd0e2b65afefdda2ddd08f9ee51a3914c0616b38).

@ales-erjavec, @markotoplak, can we increase the oldest supported version to 0.12?

@ales-erjavec, can you add it to this PR?

**Comment by ales-erjavec** [comment]
> ... can we increase the oldest supported version to 0.12?

I have reset the palette instead so that wont be necessary.

**Pull request merged**